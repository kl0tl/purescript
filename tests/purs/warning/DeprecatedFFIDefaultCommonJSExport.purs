-- @shouldWarnWith DeprecatedFFIDefaultCommonJSExport
module Main where

foreign import default :: forall a. a
