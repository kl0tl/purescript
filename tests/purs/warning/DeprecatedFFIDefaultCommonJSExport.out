Warning found:
at tests/purs/warning/DeprecatedFFIDefaultCommonJSExport.purs:2:1 - 4:38 (line 2, column 1 - line 4, column 38)

  In the FFI module for [33mMain[0m:

    CommonJS exports named [33mdefault[0m are deprecated and wonâ€™t be supported in the future.



See https://github.com/purescript/documentation/blob/master/errors/DeprecatedFFIDefaultCommonJSExport.md for more information,
or to contribute content related to this warning.

